tests/api_resources/test_file.py

```diff
@@ -38,14 +42,19 @@ def test_is_creatable(self, setup_upload_api_base, request_mock):
             file=test_file,
             file_link_data={"create": True},
         )
-        request_mock.assert_api_base(stripe.upload_api_base)
-        request_mock.assert_requested(
-            "post",
-            "/v1/files",
-            headers={
-                "Content-Type": "multipart/form-data; boundary=1234567890"
-            },
+        method, url, headers, body = hc.request_with_retries.call_args[0]
+        assert method == "post"
+        assert url.endswith("/v1/files")
+        assert (
+            headers["Content-Type"]
+            == "multipart/form-data; boundary=1234567890"
         )
+        parts = body.split(b"--1234567890")
+        assert len(parts) == 5
+        assert parts[0] == b""
+        assert parts[1].find(b'name="purpose"') > 0
```

Comment:
`assert b'name="purpose"' in parts[1]` ? here and elsewhere

Reply:
Done!